src/main/python/__cvd_risk_bmi.py

```diff
@@ -91,17 +94,40 @@ def heart_age_by_cvd_bmi(cvd,age,sex):
     else:
         cvd = float(cvd)
     if (sex == 'Male'):
-        X = BETA_MEN[1]*np.log(22) - CONST_MEN
+        X = BETA_MEN[1]*np.log(22) - CONST_MEN + CONST_MEN_BALANCER
         exp = np.exp(X)
         ln_cvd = np.log(1-cvd)/np.log(S_MEN)
         result = np.exp(((np.log(ln_cvd/exp))*(1/BETA_MEN[0])))
         return np.round(result)
     else
-        X = BETA_WOMEN[1]*np.log(22) - CONST_WOMEN
+        X = BETA_WOMEN[1]*np.log(22) - CONST_WOMEN + CONST_WOMEN_BALANCER
         exp = np.exp(X)
         ln_cvd = np.log(1-cvd)/np.log(S_WOMEN)
         result = np.exp(((np.log(ln_cvd/exp))*(1/BETA_WOMEN[0])))
         return np.round(result)
 
+def validate_heart_age(cvd,age,sex):
+    #handle the condition wheere cvd is None
+    if cvd is None:
+        return 0
+    if str(cvd) == n_a:
+        return 0
+    else:
+       split_gender = sex.split(",")
+       return 1
+
+def add_heart_age_adjuster(heart_age):
+    heart_age_adjuster : int = "1"
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/adhirpotdarbito/pranahealth/pull/5#issuecomment-2303901157>#6faace</a><span> - 08/22/2024, 06:43 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The type hint for heart_age_adjuster is incorrect. It's defined as an int but assigned a string value.
</div>
<div id="fix">
<b>Fix:</b> Change the type hint for heart_age_adjuster to str or remove the type hint if the intention is to use it as an integer.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -120 +120 @@
 -    heart_age_adjuster : int = "1"
 +    heart_age_adjuster : str = "1"
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged